src/controller/store/oneCategory.js

```diff
@@ -0,0 +1,10 @@
+const {oneCategory} = require('../../database/queries/oneCategory');
+const oneCat = (req,res) =>{
+    const categoryName = req.params.category;
```

Comment:
Use destructuring to extract categoryName directly from req.params is another valid approach, and it can make the code more concise and readable.

```js
const { categoryName } = req.params;
```